content/swift/concepts/dictionaries/terms/forEach/foreach.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'forEach'
+Description: 'The forEach method allows iteration over each key-value pair in a Swift dictionary.'
+Subjects:
+  - 'iOS'
+Tags:
+  - 'ForEach'
+  - 'Dictionary'
+  - 'Loops'
+  - 'Collections'
+  - 'iOS'
```

Comment:
```suggestion
  - 'iOS'
```
The `iOS` named tag is not present in the tags.md list. You can add it if you want it to be included in the file.

Reply:
It is included in tags.md. I double-checked just to be sure, and it's there.